octo/model/components/tokenizers.py

```diff
@@ -202,7 +203,7 @@ def __call__(
             assert self.proper_pad_mask, "Cannot skip unless using proper pad mask."
             return None
 
-        if not isinstance(tasks["language_instruction"], jax.Array):
+        if not isinstance(tasks["language_instruction"], (jax.Array, np.ndarray)):
```

Comment:
When is this ever an np array?

Reply:
@SudeepDasari I think this got added in the new action head PR?